server/src/buildSchema.ts

```diff
@@ -0,0 +1,392 @@
+// This file has been generated from https://raw.githubusercontent.com/rescript-lang/rescript-compiler/master/docs/docson/build-schema.json
```

Comment:
Would you add a note on how one can regenerate this if it's updated?

Reply:
Hmmm, now that I think about it, quicktype ignores the required fields so I corrected them by hand. Is it ok if I add https://github.com/bcherny/json-schema-to-typescript as a dev-dependency instead and add it to the npm scripts?

Reply:
Okay, will just strip down the definitions then.

Reply:
Will also remove all the comments, since it's a lot of noise.